tdrs-backend/tdpservice/search_indexes/management/commands/clean_and_reparse.py

```diff
@@ -20,16 +20,16 @@
 class Command(BaseCommand):
     """Command class."""
 
-    help = "Delete and re-parse a set of datafiles. All re-parsed data will be moved into a new set of Elastic indexes."
+    help = "Delete and reparse a set of datafiles.."
 
     def add_arguments(self, parser):
         """Add arguments to the management command."""
         parser.add_argument("-q", "--fiscal_quarter", type=str, choices=["Q1", "Q2", "Q3", "Q4"],
-                            help="Re-parse all files in the fiscal quarter, e.g. Q1.")
-        parser.add_argument("-y", "--fiscal_year", type=int, help="Re-parse all files in the fiscal year, e.g. 2021.")
-        parser.add_argument("-a", "--all", action='store_true', help="Clean and re-parse all datafiles. If selected, "
+                            help="Reparse all files in the fiscal quarter, e.g. Q1.")
+        parser.add_argument("-y", "--fiscal_year", type=int, help="Reparse all files in the fiscal year, e.g. 2021.")
+        parser.add_argument("-a", "--all", action='store_true', help="Clean and reparse all datafiles. If selected, "
                             "fiscal_year/quarter aren't necessary.")
-        parser.add_argument("-n", "--new_indices", action='store_true', help="Move re-parsed data to new Elastic "
+        parser.add_argument("-n", "--new_indices", action='store_true', help="Move reparsed data to new Elastic "
                             "indices.")
         parser.add_argument("-d", "--delete_indices", action='store_true', help="Requires new_indices. Delete the "
                             "current Elastic indices.")
```

Comment:
Not having these selected results in failures and inconsistency, we should remove them as parameters and just always have these executed as true, as far as my understanding.

Reply:
Can you digress on the failures you saw? In my opinion, the only time we should specify these options is when we want to reindex the entire DB and get elastic consistent. I think having elastic indexes dedicated to a subset of the data does not make sense. In the future they only want to reparse single files. In that case we would create 21 new indexes to store a single file. We will overload elastic very quickly doing that. With that said, I would be open to removing the availability of the options in favor of forcing new indices if and only if `-a` is specified.

Reply:
I no longer have the errors in my shell scrollback, but it was to the effect that running -a on different environments was causing drift between ES and postgres. Attempting to recreate this behavior with the deployed code from #3136 no longer shows the behavior as you alluded to [here](https://github.com/raft-tech/TANF-app/pull/3126#issuecomment-2296553787). 

Given the discrepancies in behavior, we should merge that PR into this one for simplicity. Yes, let's remove the availability for those when using -a and force new indices. After this change, I'll be good to send this over to QASP.